develop/devguide/Automation/Howto/BP_Implementing_IAS_in_Regression_Tests.md

```diff
@@ -23,15 +23,15 @@ When implementing interactive Automation scripts that may be eligible to be test
 
 ## Use JSON parameters for user decisions
 
-Introduce a script input parameter that accepts a JSON structure containing the decisions a user will need to make during the interaction. This JSON can include various parameters like user inputs, options, or configurations required to execute the script.
+Introduce a script input parameter that accepts a JSON structure containing the decisions a user would make during the interaction. This JSON can include various parameters like user inputs, options, or configurations required to execute the script.
 
 By encapsulating these decisions in JSON format, the script gains flexibility and can easily adapt to different scenarios without requiring code modifications.
 
 **Example**:
 
 Below, you will find three startup input parameters required to trigger the script.
 
-This input parameter contains the Interactive Input JSON, representing the decisions a user normally makes during the interaction:
+The 4th input parameter contains the Interactive Input JSON, representing the decisions a user normally makes during the interaction:
 
```

Comment:
@SkylineTimothyVanPoucke, I find this confusing. We first introduce three startup input parameters without showing them, and then also mention a fourth first. I assume the three startup input parameters mentioned are the three images named "Deserialize", and that the 4th input parameter is the first image named "Interactive Input Json"? Perhaps we should move around the text to make things more clear:

This input parameter contains the Interactive Input JSON, representing the decisions a user normally makes during the interaction:

![Interactive Input Json](~/develop/images/InteractiveInputJson.png)

When we read out these parameters, we try to deserialize this JSON. If successful, we run the script non-interactively; otherwise, we trigger the script interactively. Below, you will find three additional startup input parameters required to trigger the script.

![Deserialize](~/develop/images/Deserialize.png)
![Deserialize](~/develop/images/Deserialize2.png)
![Deserialize](~/develop/images/Deserialize3.png)